recipes-devtools/partition-utils/qcom-gen-partitions-tool_1.0.bb

```diff
@@ -0,0 +1,21 @@
+SUMMARY = "ptool compliant partition generation utility"
+DESCRIPTION = "Generates partition.xml in ptool suitable format"
+SECTION = "devel"
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/${LICENSE};md5=550794465ba0ec5312d6919e203a55f9"
+
+RDEPENDS:${PN} += "python3-xml"
+
+SRC_URI = "file://gen_partition.py"
```

Comment:
Instead of having this script on the layer it should be better to move it to the partioning_tool.git repo if possible and provide it as an utility.

Reply:
Yeah, asking to host partioning_tool under this same github org, once approved we can merge this script there as well.